app/src/main/java/com/github/se/travelpouch/ui/home/OnboardingScreen.kt

```diff
@@ -98,9 +120,33 @@ fun OnboardingPageContent(page: OnboardingPage, modifier: Modifier = Modifier) {
         Text(
             page.description,
             fontSize = 16.sp,
-            modifier = Modifier.testTag("OnboardingDescription"))
+            modifier = Modifier.testTag("OnboardingDescription"),
+            textAlign = androidx.compose.ui.text.style.TextAlign.Center)
+      }
+}
+
+// Progress indicator to show the current page in the onboarding process
+@Composable
+fun ProgressIndicator(currentPage: Int, totalPages: Int, modifier: Modifier = Modifier) {
```

Comment:
It's wonderful but a little documentation could be helpful for maintainability purposes.